.travis.yml

```diff
@@ -3,7 +3,9 @@ cache: npm
 notifications:
   email: false
 node_js:
+  - 10.12
   - 10
+  - 12.0
```

Comment:
is this 12.0 needed since we have 12 below it?

Reply:
12.0 is specifically 12.0, while 12 is the latest 12.x